### PR TITLE
V0.2.0

### PR DESCRIPTION
**Pull request by matthdsm**
- Automated and updated gender prediction
- deprecated gender split references, unified references
- code clean up

WARNING:
- Breaks backwards compatibility with NPZ files generated with previous versions. A conversion script will be provided to convert older NPZ files.

### CODE REVIEW

File: additional/fix.convert.npz.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+# functions (newref, predict) should be re-run.
+
+# Usage: python fix.convert.npz.py input.npz output.npz
+
+import sys
+import numpy as np
+
+_, in_npz, out_npz = sys.argv
+
+def toolConvert():
```
Comment (matthdsm) [inline additional/fix.convert.npz.py]: What are you trying to do here?

Comment (leraman) [reply additional/fix.convert.npz.py]: .npz files contain references to functions when they are badly generated. This function needs to be present to enable conversion of original WISECONDOR files

Comment (matthdsm) [reply additional/fix.convert.npz.py]: So the return value is `1` or does the `1` return as `True`?

Comment (leraman) [reply additional/fix.convert.npz.py]: It returns nothing -- but it doesn't really matter: something in WISECONDOR's .npz files calls this function, but its result is not used.

Comment (leraman) [reply additional/fix.convert.npz.py]: The 1 is there for syntax reasons

**Pull request merged**